src/pages/sandbox/case-studies-core-a.html

```diff
@@ -0,0 +1,65 @@
+---
+title: Case Studies A
+layout: toolkit.default
+section:
+  heading: Our work
+  subheading: Oh, the things we've built!
+notes: |
+  Work overview page.
+stylesheets:
+  - sandbox/sara-wip
+---
+
+{{! Intro }}
+<div class="u-containProse u-spaceEnds6 u-spaceItems1">
+  <p class="TextBlock">
+    We've built responsive web sites, JavaScript applications, hybrid apps, you name it—if it's mobile, we've built it.
+  </p>
+  <p>
+    We've worked on high-profile projects ranging from Walmart's responsive web redesign to the official Obama '08 iPhone App.
+  <p>
+</div>
+
+{{! Logos }}
+<div class="u-release u-bgGray u-spaceEnds6 u-padEnds3 u-md-padEnds6 u-padSides1">
+  <div class="Grid Grid--alignCenter Grid--alignMiddle u-containSpread">
+    {{#each drizzle.data.case-studies.contents.logos}}
```

Comment:
I don't know if we're using an up-to-date enough version of Drizzle. 😕 

It's been difficult to prioritize upgrading that stuff, but we probably should at some point.